cli/cmd/tapRunner.go

```diff
@@ -405,7 +405,7 @@ func watchApiServerEvents(ctx context.Context, kubernetesProvider *kubernetes.Pr
 			case "FailedScheduling", "Failed":
 				logger.Log.Errorf(uiUtils.Error, fmt.Sprintf("Mizu API Server status: %s - %s", event.Reason, event.Note))
 				cancel()
-				break
```

Comment:
why the break here removed? 
don't we want to stop the loop after it "FailedScheduling" or "Failed"